### ISSUE TITLE
Can't focus on Text with TalkBack in Android Q

### ISSUE THREAD

**Issue by alanrussian** [flutter/flutter#30753]
When I run this app, I can focus on the Text widget on my Android O device but not my Android Q device with TalkBack. 

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  final String title;

  const MyHomePage({Key key, this.title}) : super(key: key);

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return MyScaffold();
  }
}

class MyScaffold extends StatelessWidget {
  const MyScaffold({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(child: Text('hi')),
    );
  }
}
```

**Comment by jonahwilliams** [comment]
I'm downloading the beta as we speak...

**Comment by jonahwilliams** [comment]
I can reproduce the issue using the Flutter Gallery about dialog

**Comment by jonahwilliams** [comment]
Some interesting logs from TallBack:

```
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.472 10447 10447 I chatty  : uid=10116(com.google.android.marvin.talkback) identical 2 lines
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.472 10447 10447 I chatty  : uid=10116(com.google.android.marvin.talkback) identical 1 line
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.472 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=false
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: checkChildren=false and isAccessibilityFocusableInternal()=true
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Don't focus, failed all focusability tests
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Speaking, has text
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Does have actionable speaking children (child 0)
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Speaking, has non-actionable speaking children
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Does have actionable speaking children (child 0)
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Speaking, has non-actionable speaking children
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Does have actionable speaking children (child 0)
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Speaking, has non-actionable speaking children
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Does have actionable speaking children (child 0)
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Speaking, has non-actionable speaking children
04-08 20:36:22.473 10447 10447 V AccessibilityNodeInfoUtils: Focus, node is focusable and has something to speak
```

**Comment by jonahwilliams** [comment]
There is nothing obviously wrong with our node tree - it seems like the text here is marked focusable

https://gist.github.com/jonahwilliams/4a6026e5833fd949ee34602d4f68a831

**Comment by jonahwilliams** [comment]
cc @mklim

**Comment by mklim** [comment]
Root cause is a bug in Q. It's been fixed upstream.

**Comment by Hixie** [comment]
@alanrussian Have you been able to verify if this is fixed?

**Comment by alanrussian** [comment]
I don't see this issue any longer in the Q build I'm currently on.
